dags/openshift_nightlies/tasks/install/rosa/defaults.json

```diff
@@ -0,0 +1,63 @@
+{
+    "orchestration_user": "",
+    "orchestration_host": "",
+    "sshkey_token": "",
+    "openshift_client_location": "",
+    "openshift_base_domain": "perfscale.devcluster.openshift.com",
+    "openshift_cluster_name": "",
+    "openshift_install_release_image_override": "",
+    "openshift_install_binary_url": "",
+    "openshift_install_apiversion": "v1",
+    "openshift_install_ssh_pub_key_file": "/home/airflow/workspace/perf-dept/ssh_keys/id_rsa_pbench_ec2.pub",
+    "openshift_install_pull_secret": "",
+    "openshift_install_quay_registry_token": "",
+    "openshift_install_image_registry": "registry.svc.ci.openshift.org",
+    "openshift_install_registry_token": "",
+    "openshift_install_installer_from_source": false,
+    "openshift_install_installer_from_source_version": "",
+    "gopath": "/root/.go",
+    "openshift_cidr": "10.128.0.0/10",
+    "openshift_machine_cidr": "10.0.0.0/16",
+    "openshift_service_network": "172.30.0.0/16",
+    "openshift_host_prefix": "22",
+    "openshift_network_type": "",
+    "openshift_post_install_poll_attempts": "600",
+    "openshift_toggle_infra_node": true,
+    "openshift_toggle_workload_node": true,
+    "dittybopper_enable": true,
+    "kubeconfig_path": "",
+    "cerberus_config_path": "/root/cerberus.yml",
```

Comment:
Cerberus was deprecated out so We shouldn't need these vars anymore.

Reply:
I have just copied the defaults file from the cloud platform.
Im not using almost any of them during the installation of rosa, but Im not sure to remove them, as I dont know if they are required during the benchmark execution.

@whitleykeith can you confirm that if I remove any non used var from the installation defaults.json, it wont break anything?

Reply:
yeah you can remove them. The install vars are only for install and your defaults.json is only for your specific install class

Reply:
all removed